### ISSUE TITLE
Add more sensors

### ISSUE THREAD

**Issue by DavidMStraub** [DavidMStraub/homeassistant-homeconnect#38]
E.g. oven core temperature.

Is there a list of sensors supported by the API somewhere??

**Comment by nic0dk** [comment]
looked at their API options for Coffee machine, what i miss is the level status of water and beans, doesnt seems like they have it :(

**Comment by Krocko** [comment]
@nic0dk 
https://developer.home-connect.com/docs/monitoring/appliance_events

**Comment by nic0dk** [comment]
didnt see them, that is useful indeed, i will love to get these level into HA :)

**Comment by DavidMStraub** [comment]
Related to #42, the question is whether something like this would be better represented by a sensor or by a custom event in HA. These events have three possible states: Off, Present, Confirmed. https://developer.home-connect.com/docs/monitoring/eventpresentstate. So either one makes them a sensor with string value or fires a custom event once it becomes "Present" and another one onceit becomes "Confirmed". I tend to think events make more sense.

**Comment by Krocko** [comment]
I vote for the sensor.

**Comment by nic0dk** [comment]
i like sensors because you will have the state showed on a card until it have gotten an new state.

im not sure about events, i think it as a event that happen and the only place to find the event is under "logbook" and you can make automation on it?

or is you HA custom events different compare to the build-in logbook events?

**Comment by Krocko** [comment]
@DavidMStraub is it also possible for you, to add more coffee programs?
See here:
https://github.com/thoukydides/homebridge-homeconnect

**Comment by DavidMStraub** [comment]
> im not sure about events, i think it as a event that happen and the only place to find the event is under "logbook" and you can make automation on it?

Yes, it would be possible to use them in automations (I guess this is the main use case).

> is it also possible for you, to add more coffee programs?

It's quite simple, there is a list of programs in the source code. I'd be happy about a PR.

**Comment by emilp333** [comment]
Hi David,

I have a request for a new sensor. Since every program is its own sensor I think it makes sense to have an overall sensor combining the program statuses. Meaning you have a sensor saying that a program is running at all. Something like in the attribute of "operation_state".

I can of course makes my own template sensor to evaluate it but I think it makes more sense to be in the component it self.
